CHANGELOG.rst

```diff
@@ -1,6 +1,51 @@
 Changelog
 =========
 
+0.5.0 (2018-08-17)
+------------------
+
+MLflow 0.5.0 includes some major improvements, including Keras and PyTorch first-class support as models, SFTP support as an artifactory, an improved scatterplot visualization to compare runs, and a more complete Python SDK for low-level experiment and run management.
+
+Breaking changes:
+
+- The Tracking API has been split into two pieces, a "basic logging" and a "tracking service" API. The "basic logging" API deals with logging metrics, parameters, and artifacts to the currently-active active run, and is accessible in ``mlflow`` (e.g., ``mlflow.log_param``). The tracking service API allow managing experiments and runs (especially historical runs) and is available in ``mlflow.tracking``. The tracking service API will look analogous to the upcoming R and Java Tracking Service SDKs. Please be aware of the following breaking changes:
```

Comment:
`a "basic logging" and a "tracking service" API` -> `a "basic logging" API and a "tracking service" API`